CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
-* HEAD
-  * Updated dependencies ->
-    * Clojure 1.9.0
-    * pedestal-service 0.5.5
-    * liberator 0.15.2
-    * ring-core 1.7.1
-    * cheshire 4.8.1
-    * schema 1.1.10
-    * clj-time 0.15.1
-    * transit 0.8.313
-* 0.7.1?
+* 0.7.1-SNAPSHOT - Unreleased
```

Comment:
🤔 Maybe a note about this now supporting Java 11?

Reply:
@sduckett Excellent idea :+1: